src/components/menu/menu-gestures.ts

```diff
@@ -88,24 +98,24 @@ export class MenuContentGesture extends SlideEdgeGesture {
   }
 
   getElementStartPos(slide: SlideData, ev: any) {
-    if (this.menu.side === 'right') {
+    if (this.menu.side === 'right' || (this.menu.side === 'start' && this.plt.isRTL()) || (this.menu.side === 'end' && !this.plt.isRTL())) {
       return this.menu.isOpen ? slide.min : slide.max;
+    } else {
+      return this.menu.isOpen ? slide.max : slide.min;
     }
-    // left menu
-    return this.menu.isOpen ? slide.max : slide.min;
   }
 
   getSlideBoundaries(): {min: number, max: number} {
-    if (this.menu.side === 'right') {
+    if (this.menu.side === 'right' || (this.menu.side === 'start' && this.plt.isRTL()) || (this.menu.side === 'end' && !this.plt.isRTL())) {
       return {
         min: -this.menu.width(),
         max: 0
       };
+    } else {
```

Comment:
let's only include the relevant changes in this PR. why did you refactor the "otherwise" condition?